Auth0/CredentialsManager.swift

```diff
@@ -84,10 +84,37 @@ public struct CredentialsManager {
     /// Clear credentials stored in keychain
     ///
     /// - Returns: if credentials were removed
+    @available(*, deprecated, message: "use clearAndRevokeToken(callback) instead")
     public func clear() -> Bool {
         return self.storage.deleteEntry(forKey: storeKey)
     }
 
+    /// Revokes the refresh token and, if successful, the credentials are cleared. Otherwise,
+    /// the credentials are not cleared and an error is raised through the callback.
+    ///
+    /// - Parameter callback: callback with an error if the refresh token could not be revoked
+    public func clearAndRevokeToken(_ callback: @escaping (CredentialsManagerError?) -> Void) {
```

Comment:
might be better to be explicit here in the naming `clearAndRevokeRefreshToken` although feels a bit wordy. I am wondering if this should simply be `revoke` which performs both actions. let's discuss internally

Reply:
This has now been renamed to `revoke`.